.github/workflows/e2e.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v2
       with:
-        go-version: "1.16.15"
```

Comment:
Right now, the 1.17.x is unsupported. 
Just wonder that it will be great to use current stable version like 1.18.x or 1.19.x.

Reply:
I would minimize the changes (so keep 1.17), as we are already breaking the usual promise of the patch versions (due to etcd long lifecycle).

Reply:
> minimize the changes

This was exactly what I was thinking. In the beginning, I bumped to go1.19.4 directly, but after second thought, I downgrade it to 1.17.13. 

If all maintainers agree to bump 1.19.x, we can do it in a separate PR.